### PR TITLE
OCE table

### PR DESCRIPTION
**Pull request by kalexand-rh**
@vikram-redhat, is this what you want?

I didn't edit the text, but it's a little inconsistent. It also doesn't meet the style guide. I can edit the text, if you want.
image

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@kalexand-rh thanks for doing this. Yes, this is great and yes please go ahead and edit the text for consistency.

**Comment by kalexand-rh** [comment]
Great! I'll clean up the text tomorrow. If you know what branches you want it on, I can merge it, too.

**Comment by kalexand-rh** [comment]
@vikram-redhat, PTAL at the edits. The source had some acronyms that I wasn't familiar with, and I made a couple of informed guesses about the capitalization for related projects.

File: welcome/oce_about.adoc [hunk 1]
```diff
@@ -0,0 +1,211 @@
+[[about_oce]]
+= About OCE
```
Comment (kalexand-rh) [inline welcome/oce_about.adoc]: I think this needs a different title.

File: welcome/oce_about.adoc [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+
```
Comment (kalexand-rh) [inline welcome/oce_about.adoc]: We need a little context to introduce the table.

File: welcome/oce_about.adoc [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+| Quay integration
+| X
+| X
+
+| Container Native Virtualization for virtual machines
+| X
+| X
+
+| Multi-cluster federation
```
Comment (kalexand-rh) [inline welcome/oce_about.adoc]: Is this correct? Where's the doc for it?

File: welcome/oce_about.adoc [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+2+h|Developer experience
+h| 
+h| X
+
+.4+|
+| Integrated CI/CD pipelines
+| 
+| X
+
+| Source to image container factory
```
Comment (kalexand-rh) [inline welcome/oce_about.adoc]: Is "Source to image container factory" a product I don't know about? Please double-check the capitalization.

File: welcome/oce_about.adoc (hunk 2 above)
Comment (vikram-redhat) [reply welcome/oce_about.adoc]: I am using the intro section of the source document for this. It is markety, so feel free to change.

"Red Hat OpenShift Container Engine (OCE) is a new product offering from Red Hat and can be used if you wish to use OpenShift as a production platform for launching containers without leveraging the more advanced features in OpenShift Container Platform (OCP). OCE is the same product as OCP, in terms of download, installation, and general usage. 

OCE entitles you to use a subset of the features found in OCP and the table here highlights the differences between these two platforms."

File: welcome/oce_about.adoc (hunk 3 above)
Comment (vikram-redhat) [reply welcome/oce_about.adoc]: I don't think this has been documented or on the cards yet.

File: welcome/oce_about.adoc (hunk 4 above)
Comment (vikram-redhat) [reply welcome/oce_about.adoc]: This should be Source-to-Image Container factory. Not sure if factory needs to be capitalized.

**Comment by vikram-redhat** [comment]
@kalexand-rh this is to be merged to 3.11 branch only (I apologize, I should have clarified as I see this is open against the master branch). You can merge it in master, and put the distro for enterprise only and then CP to 3.11.

**Comment by vikram-redhat** [comment]
xref: https://github.com/openshift/openshift-docs/pull/12103. 

When this merges, #12103 can be merged as part of 3.11 release.

**Comment by kalexand-rh** [comment]
@vikram-redhat, I've made updates based on your changes and am merging. If anything looks amiss, please make more changes.

**Pull request merged**